### PR TITLE
CORTX-28904:changed rgw external ports to 80 & 443

### PR DESCRIPTION
**Pull request by lakshita-jain**
Signed-off-by: Lakshita Jain <lakshita.jain@seagate.com>

### CODE REVIEW

**Review by ujjwalpl — APPROVED** [review]

File: k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-platform/templates/cortx-io-svc.yaml [hunk 1]
```diff
@@ -11,11 +11,11 @@ spec:
     - name: http-port80
       protocol: TCP
       port: 80
-      targetPort: 8000
+      targetPort: 80
```
Comment (tanujashinde0405) [inline k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-platform/templates/cortx-io-svc.yaml:14]: change this to 22751

File: k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-platform/templates/cortx-io-svc.yaml [hunk 2]
```diff
@@ -11,11 +11,11 @@ spec:
...
     - name: https-port443
       protocol: TCP
       port: 443
-      targetPort: 8443
+      targetPort: 443
```
Comment (tanujashinde0405) [inline k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-platform/templates/cortx-io-svc.yaml:18]: change this to 23001

File: k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-server/templates/cortx-server-svc.yaml [hunk 3]
```diff
@@ -37,8 +37,8 @@ spec:
     - name: cortx-rgw-http
       protocol: TCP
       port: 80
```
Comment (tanujashinde0405) [inline k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-server/templates/cortx-server-svc.yaml:39]: change this to 22751

File: k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-server/templates/cortx-server-svc.yaml [hunk 4]
```diff
@@ -37,8 +37,8 @@ spec:
...
-      targetPort: 8000
+      targetPort: 22751
     - name: cortx-rgw-https
       protocol: TCP
       port: 443
```
Comment (tanujashinde0405) [inline k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-server/templates/cortx-server-svc.yaml:43]: change this to 23001

**Review by tanujashinde0405 — CHANGES_REQUESTED** [review]

**Review by nitin-seagate — APPROVED** [review]
LGTM

File: k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-platform/templates/cortx-io-svc.yaml (hunk 2 above)
Comment (lakshita-jain) [reply k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-platform/templates/cortx-io-svc.yaml]: changes done in the new commit

**Review by tanujashinde0405 — APPROVED** [review]

**Pull request merged**